packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -48,6 +51,28 @@ typedef SetTextHandler = void Function(String text);
 /// Returned by [SemanticsConfiguration.getActionHandler].
 typedef SemanticsActionHandler = void Function(Object? args);
 
+/// The semantics merge rule generated from [SemanticsConfiguration.merger].
+class SemanticsMergeRule {
+  /// The child semantics configuration that are merging upward.
+  final List<SemanticsConfiguration> mergeUp = <SemanticsConfiguration>[];
+
+  /// The groups of child semantics configuration that want to merge together
+  /// and form a child [SemanticsNode].
+  ///
+  /// If the [SemanticsConfiguration] in the group is a semantics boundary or
```

Comment:
Oh, do you mean _all_ the semanticsConfigurations that are boundaries or conflict will be excluded from the merge? Maybe say that: All the [SemanticsConfiguration]s in a given group that are either semantics boundaries or are conflicting with other [SemanticsConfiguration]s of the same group will be excluded from the sibling merge group and form independent semantics nodes as usual.